packages/rich-text/stories/RichTextEditor.stories.tsx

```diff
@@ -301,9 +300,9 @@ const DemoRichTextEditor = () => {
 
 export const Default: Story = {
   parameters: {
-    controls: { hideNoControlsWarning: true }
+    controls: { hideNoControlsWarning: true },
   },
   render: () => {
     return <DemoRichTextEditor />;
-  }
+  },
```

Comment:
unrelated but we should have some auto formatting rules for this repo, a lot of the code changes are comma's everywhere 🙈